roles/docker/tasks/Fedora.yml

```diff
@@ -0,0 +1,16 @@
+---
+- name: Add docker gpg key
+  rpm_key:
+    key: "{{ docker_gpg_key_url }}"
+
+- name: Add stable docker repo
+  command: "dnf config-manager --add-repo {{ docker_repo_url }}"
```

Comment:
Did you try to use the `yum_repository` module for this? DNF still uses `/etc/yum.repos.d`